### PR TITLE
Checkboxes

### PR DESCRIPTION
**Pull request by ngprnk**
# Changelog 
1. Input boolean 
2. Basecheckbox 
3. Checkbox - single, multiple / select. Boxes have icons to differentiate types.

### CODE REVIEW

File: src/views/components/input/checkbox-2/InputCheckbox.js [hunk 1]
```diff
@@ -7,32 +7,35 @@ class InputCheckbox extends Component {
   static defaultProps = {
```
Comment (conorpirie) [inline src/views/components/input/checkbox-2/InputCheckbox.js]: can this directory name be changed to `checkbox`?

Comment (conorpirie) [reply src/views/components/input/checkbox-2/InputCheckbox.js]: or removed entirely if it's superfluous

Comment (conorpirie) [reply src/views/components/input/checkbox-2/InputCheckbox.js]: getting this error when running. can you resolve this and the above issues then I can test+merge

```ERROR in ./config/routes/routes.web.js
Module not found: Error: Can't resolve '../../views/components/input/checkbox-5' in '/Users/conorpirie/projects/genny/alyson-v7/src/config/routes'
 @ ./config/routes/routes.web.js 1:153-203```

Comment (ngprnk) [reply src/views/components/input/checkbox-2/InputCheckbox.js]: Done.

**Pull request merged**